### PR TITLE
Add strawman test about freshness

### PR DESCRIPTION
**Pull request by dthaler**
Based on discussion at IETF about including a very high level summary,
and text in the arch docs, draft-birkholz-rats-reference-interaction-model,
and draft-birkholz-rats-tuda.

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -327,7 +327,38 @@ implicitly trusted is often referred to as a Root of Trust.
...
+that the claims were signed at that time, as long as it has some assurance that
+the timestamp is correct.  This typically requires additional claims about
+the signer's time synchronization mechanism in order to provide such assurance.
+
+In either approach, it is important to note that the actual values in claims
+might have been generated long before the claims are signed.  If so, it is the
+signer's responsibility to ensure that the values are still correct when they
+are signed.  For example, values might have been generated at boot, and then
+used in claims as long as the signer can guarantee that they cannot have changed
+since boot.
```
Comment (henkbirkholz) [inline draft-ietf-rats-architecture.md]: Boot-Cycle is a good example I think (but only before the entropy of user-space kicks in. There are other problems regarding the "stability" of these "early boot measurements" today, but that is beside the point here, I think). But still leaves the question open how such a guarantee looks like.

In summary there seem to be four relevant points on time here:

1.) creation of the value in the attestedEE
2.) measurement of the value in the attestingEE to create the Claim value
3.) creation of the evidence
    * with creation of the nonce _before_ the creation of the evidence (although I am still unsure why there is an emphasis on that), or
    * with retrieval of a trustworthy timestamp as close as possible before the creation of the evidence
4.) conveyance/retrieval of the evidence to/by an evidence sink (in general the Verifier or a relaying entity)

Which of these (at least) four points in time are now actually vital wrt to expressing the freshness of the thing conveyed? Maybe this should go into the 2nd item of this section after "an elaboration of the concept of freshness"?

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -327,7 +327,38 @@ implicitly trusted is often referred to as a Root of Trust.
 
 # Freshness
 
-    <this section can include some high-level content from draft-birkholz-rats-reference-interaction-model>
+It is important to prevent replay attacks where an attacker replays
```
[the remark this answers is not in the document]
Comment (dthaler) [reply draft-ietf-rats-architecture.md]: Maybe we change the section title to "Replay Prevention"?

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -327,7 +327,38 @@ implicitly trusted is often referred to as a Root of Trust.
...
+old Evidence or an old Attestation Result that is no longer correct.
+To do so, some mechanism of ensuring that the Evidence and Attestation
+Result are fresh, meaning that there is some degree of assurance
```
[the remark this answers is not in the document]
Comment (dthaler) [reply draft-ietf-rats-architecture.md]: I have no preference.  No specific intent.

File: draft-ietf-rats-architecture.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dthaler) [reply draft-ietf-rats-architecture.md]: (changed section title)

[the remark this answers is not in the document]
Comment (dthaler) [reply draft-ietf-rats-architecture.md]: Changed it back and added a sentence into the security considerations section about replay prevention.

**Pull request merged**